hadoop-tools/hadoop-azure/src/site/markdown/abfs.md

```diff
@@ -665,6 +665,79 @@ performance issues.
 Config `fs.azure.enable.check.access` needs to be set true to enable
  the AzureBlobFileSystem.access().
 
+### <a name="authconfigoptions"></a> Auth Options
+`fs.azure.account.key`: To set the account access key. Access keys can be used to authenticate the requests made from the ABFS driver to the Azure storage account.
```

Comment:
account key config is already explained with an example in the above shared key section. Please cross check with the details above and if needed more comments can be added in respective place.

Reply:
Comment holds for other configs too, please check.

Reply:
Done